File: challenges/2-tabuada/c++/baka-noob/tabuada.cpp
#include <iostream>

using namespace std;

void tabuada(int numero) {

    int i, n, resultado;
    for (i = 1; i <= numero; i++) { 

        cout << "===========================" << endl;
        cout << "A tabuada do numero " << i << " é:" << endl;

        for (n = 1; n <= 10; n++) {

            resultado = i * n;
            cout << "   " << i << "*" << n << " = " << resultado << endl;
            
        }
    }
    cout << "===========================" << endl;
}

int main() {

    int numero;
    cin >> numero;
    tabuada(numero);

    return 0;
}

File: challenges/2-tabuada/php/arthurabreu00/tabuada.php
<?php

try{
    if(count($argv) !== 2 || !is_numeric($argv[1])) throw new Exception('Informe apenas um número, para exibição da tabuada');

    $tabuada = $argv[1];
    
    for ($i = 0; $i < $tabuada; $i++) 
    {
        $numero = $i+1;
        echo '-------------------------------------------------------------' . PHP_EOL;
        echo "A tabuada do número {$numero}:" . PHP_EOL;
        echo '-------------------------------------------------------------' . PHP_EOL;
        for ($multiplicador = 1; $multiplicador <= 10; $multiplicador++) {   
            $resultado = $numero * $multiplicador;
            echo "{$numero} x {$multiplicador} = {$resultado}" . PHP_EOL;
        }
    }

} catch (Exception $e){
    echo $e->getMessage() . PHP_EOL;
}